src/compas_ghpython/components/Compas_ToRhinoGeometry/code.py

```diff
@@ -4,12 +4,12 @@
 
 from ghpythonlib.componentbase import executingcomponent as component
 
-from compas.scene import SceneObject
+from compas.scene import sceneobject_factory
 
 
 class CompasToRhinoGeometry(component):
     def RunScript(self, cg):
         if not cg:
             return None
 
-        return SceneObject(item=cg).draw()
+        return sceneobject_factory(item=cg).draw()
```

Comment:
@gonzalocasas @chenkasirer Please note here, with the factory pattern, you need call factory function in GH components instead of constructor